### ISSUE TITLE
Data view stylesheet not working

### ISSUE THREAD

**Issue by gamontoya** [ucsdlib/damsrepo#50]
## Issue:
Cannot access data view of the following object

- https://librarytest.ucsd.edu/dc/collection/bd4590097p/data

Note that this is on staging.

## Error:
500Error transforming metadata2017-06-12T08:53:41-0700
javax.xml.transform.TransformerException: java.lang.RuntimeException: Invalid conversion from 'java.lang.String' to 'node-set'.
com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:752)

## To do:
Check to see if a new element was introduced to the stylesheet

**Comment by lsitu** [comment]
@gamontoya It looks like there is an unknown authority record http://lib-hydratail-staging.ucsd.edu:8080/dams/api/objects/bd1654915c in the collection RDF somehow, which trigger the error.
@mcritchlow  I've updated the stylesheet to fix the error, see commit https://github.com/ucsdlib/damsrepo/commit/40ecd1d1826636ea2e1e6ca0df7c778023b11d75. It's ready for review now and we'll create a release for it by the end of the sprint if it looks good. Thanks.

**Comment by gamontoya** [comment]
@lsitu Good detective work, Longshou!

**Comment by mcritchlow** [comment]
@lsitu - The commit LGTM. Feel free to setup a PR, thanks 👍

**Comment by lsitu** [comment]
@mcritchlow There's the commit for camel integration on the develop branch and I've rebased it last Friday when merging it back from release/4.33 to avoid conflicts when we need to make any changes again. 
I am not sure what's the best way to do it in this case but PR may not work unless we want to deploy the camel integration at the mean time. Maybe we can do it the same way as what we did for last sprint to create a release branch from there and merge it back to the develop branch later?
